content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

```diff
@@ -87,18 +87,24 @@ The inventory valuation report
 ------------------------------
 
 To view the current value of all products in the warehouse, go to :menuselection:`Inventory app -->
-Reporting --> Inventory Valuation`. The records in the table are organized by product, and selecting
-a product's drop-down menu displays detailed records with the :guilabel:`Date`,
-:guilabel:`Quantity`, and :guilabel:`Total Value` of the inventory.
+Reporting --> Valuation`. The page displays detailed records of products with the :guilabel:`Date`,
+:guilabel:`Quantity`, :guilabel:`Unit Value` and :guilabel:`Total Value` of the inventory.
```

Comment:
```suggestion
Reporting --> Valuation`. Doing so, reveals a page displaying detailed records of products with the
:guilabel:`Date`, :guilabel:`Quantity`, :guilabel:`Unit Value` and :guilabel:`Total Value` of the
inventory.
```
**make sure this doc mentions that this Reporting menu option is only available in 'Developer Mode' -